bin/yaml/libraries.yaml

```diff
@@ -97,6 +97,14 @@ libraries:
       build_type: cmake
       extra_cmake_arg:
         - -DBENCHMARK_ENABLE_TESTING=off
+    bmulti:
+      type: gitlab
+      method: nightlyclone
+      repo: correaa/boost-multi
+      check_file: include/multi/array.hpp
+      build_type: none
+      targets:
+        - trunk
```

Comment:
We typically put all the nightly libaries under an `if: nightly` check - I'll merge and move - thanks!

Reply:
should I add a `      if: nightly` before the `method` line? What is it for?